staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator/aggregator.go

```diff
@@ -197,7 +197,15 @@ func (s *specAggregator) updateServiceLocked(name string) error {
 		if err != nil {
 			return nil, "", err
 		}
-		return aggregator.FilterSpecByPathsWithoutSideEffects(result, []string{"/apis/"}), etag, nil
+		group := specInfo.apiService.Spec.Group
+		version := specInfo.apiService.Spec.Version
+		if group == "" {
```

Comment:
Empty group/version should never pass validation and be here as an input, are these error checks still useful?

Reply:
> Empty group/version should never pass validation and be here as an input, are these error checks still useful?

I'd probably remove them.

Reply:
removed